.htaccess

```diff
@@ -0,0 +1,16 @@
+# Apache 2.2
+<IfModule !mod_authz_core.c>
+    Order deny,allow
+    Deny from all
+    <Files ~ "(?i)^.*\.(jpg|jpeg|gif|png|bmp|tiff|svg|pdf|mov|mpeg|mp4|avi|mpg|wma|flv|webm|ico|webp|woff|woff2|ttf|eot|otf|css|js|map)$">
```

Comment:
And PHP to load express checkout scripts and paypal plus submit

Reply:
rule is modified